dlt/helpers/streamlit_app/blocks/resource_state.py

```diff
@@ -24,9 +24,9 @@ def resource_state_info(
     sources_state = pipeline.state.get("sources") or {}
```

Comment:
we need to start using helper methods to get source and resource state but OK for now